driver-core/src/main/com/mongodb/internal/connection/ClientMetadataHelper.java

```diff
@@ -37,63 +41,29 @@
  * <p>This class is not part of the public API and may be removed or changed at any time</p>
  */
 public final class ClientMetadataHelper {
-    public static final BsonDocument CLIENT_METADATA_DOCUMENT = new BsonDocument();
-
     private static final String SEPARATOR = "|";
 
-    private static final String APPLICATION_FIELD = "application";
```

Comment:
The particular one above ended up no longer used, since `application.name` is used instead. In the original file (and in the codebase in general), there are various strings/literals, sometimes repeated, that are not extracted into statics (`"java.runtime.version"`, `"unknown"`, `"os.name"`...). 

I'm not sure where exactly to draw the line - I usually avoid statics when, for example, all the usages can be placed in the same method.